app/domain/etl/item/content/content_populator.rb

```diff
@@ -0,0 +1,11 @@
+class Etl::Item::Content::ContentPopulator
```

Comment:
This code is a `one off` thing, that is migrating content as per a content schema. I am not sure it should live within our ETL processes folder.

What are your thoughts about:

1. Inline this code within the rake task, and unit test it at the rake task level
2. Move this rake task into the following folder: `/lib/tasks/one_off/001-migrate-dimension-item.rake`

Reply:
Done!